cpu_monitor:
  ros__parameters:
    temp_warn: 90.0
    temp_error: 95.0
    usage_warn: 0.90
    usage_error: 1.00
    usage_avg: true
    msr_reader_port: 7634
hdd_monitor:
  ros__parameters:
    hdd_reader_port: 7635
    num_disks: 2
    disks:  # Until multi type lists are allowed, name N the disks as disk0...disk{N-1}
      disk0:
        name: /dev/sda
        temp_warn: 55.0
        temp_error: 70.0
    usage_warn: 0.95
    usage_error: 0.99
mem_monitor:
  ros__parameters:
    usage_warn: 0.95
    usage_error: 0.99
net_monitor:
  ros__parameters:
    devices: ["*"]
    usage_warn: 0.95
ntp_monitor:
  ros__parameters:
    server: ntp.nict.jp
    offset_warn: 0.1
    offset_error: 5.0
process_monitor:
  ros__parameters:
    num_of_procs: 5
gpu_monitor:
  ros__parameters:
    temp_warn: 90.0
    temp_error: 95.0
    gpu_usage_warn: 0.90
    gpu_usage_error: 1.00
    memory_usage_warn: 0.95
    memory_usage_error: 0.99
